app/services/test_service/create.rb

```diff
@@ -0,0 +1,29 @@
+# PATH: /app/services/test_service/create.rb
+# rubocop:disable Style/ClassAndModuleChildren
+class TestService::Create < BaseService
+  attr_accessor :name, :status
+  def initialize(name, status)
+    @name = name
+    @status = status
+  end
```

Comment:
The `TestService::Create` service class is using manual validation which can be error-prone and hard to maintain. Consider using ActiveModel validations for better maintainability.